content/releases/prestashop80.md

```diff
@@ -0,0 +1,504 @@
+                </div>
+            </section>
+            <section class="section bg-light" id="multistore">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h2 class="section-title">Multistore</h2>
+                            <p class="mb-5">
+                                Many pages in the Back office have been updated for multistore compatibility, allowing merchants to choose which fields to update when in shop or group context.
+                            </p>
+                        </div>
+                        <div class="col-md-8">
+                            <p class="mt-5">
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/multistore.jpeg">
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section" id="dkim">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h2 class="section-title">DKIM signatures for emails</h2>
+                            <p>
+                                <strong>Reduce the likelihood of your shop’s emails being marked as spam</strong> by using <a href="https://en.wikipedia.org/wiki/DomainKeys_Identified_Mail">DKIM</a> signatures. This can be configured in <em>Advanced Parameters > E-mail</em>.
+                            </p>
+                        </div>
+                        <div class="col-md-8">
+                            <p class="mt-5">
+                                <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/dkim.jpeg">
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section bg-light" id="installer">
+                <div class="container">
+                    <div class="row">
+                        <div class="col">
+                            <h2 class="section-title">Installer</h2>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-6">
+                            <p>
+                                <strong>Reduce clutter by choosing which modules and themes to install</strong> during the shop’s installation process.
+                            </p>
+                            <p>
+                                <strong>Customize your distribution.</strong> During the shop’s installation, any modules and themes present in the shop’s modules and themes directories will be installed by default, unless the user chooses otherwise. This allows integrators to create customized PrestaShop builds with (or without) the modules and themes of your choosing.
```

Comment:
```suggestion
                                <strong>Customize your distribution.</strong> During the shop’s installation, any modules, themes present in the shop’s modules and themes directories will be installed by default, unless the user chooses otherwise. This allows integrators to create customized PrestaShop builds with (or without) the modules and themes of your choosing.
```